contrib/ci/fedora.sh

```diff
@@ -12,6 +12,7 @@ meson .. \
     -Dman=true \
     -Dtests=true \
     -Dplugin_dummy=true \
+    -Dplugin_modem_manager=true \
```

Comment:
Not sure it makes sense to compile under fedora under CI if it's not marked for inclusion in the spec file yet.

Reply:
It's probably worth rebasing on master since master is tracking fedora 29 and should see the newer modem manager then

Reply:
I've done the rebase, so lets see if fedora works now.